### PR TITLE
feat(mcda) - mcda feature documentation added

### PR DESCRIPTION
**Pull request by propakov**
https://kontur.fibery.io/Tasks/Task/Create-doc-for-MCDA-feature-14987

### CODE REVIEW

File: src/features/mcda/readme.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+MCDA is a basic tool to answer localization questions with GIS ("where better do or not to do something").
+Users have a need to perform a spatial analysis based on a variety of criteria.
+Previously Kontur allowed combining only 2 layers in bivariate-manager feature.
+In MCDA feature we can allow users to select as many layers as they need.
+
+## How to use
+
+To start using this feature user needs:
+
+- feature "bivariate_manager" turned on ("bivariate_manager": true)
```
Comment (IlyaIzr) [inline src/features/mcda/readme.md]: what `("bivariate_manager": true)` means?

File: src/features/mcda/readme.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+- flag "KONTUR_MCDA": true in localstorage
+
+### Activation
+
+In Draw Tools user'll see a MCDA button.
```
Comment (IlyaIzr) [inline src/features/mcda/readme.md]:
```suggestion
In Draw Tools user will see a MCDA button.
```

Comment (IlyaIzr) [reply src/features/mcda/readme.md]: Wait, isn't it in toolbar?

File: src/features/mcda/readme.md (hunk 1 above)
Comment (propakov) [reply src/features/mcda/readme.md]: that feature "bivariate_manager" is turned on =)
it's when bivariate matrix is available

File: src/features/mcda/readme.md (hunk 2 above)
Comment (propakov) [reply src/features/mcda/readme.md]: It's here 
Screenshot 2023-02-22 at 09 11 03
Yeah, you're right, it's a Toolbar. 👍

Comment (propakov) [reply src/features/mcda/readme.md]: updated

File: src/features/mcda/readme.md (hunk 1 above)
Comment (propakov) [reply src/features/mcda/readme.md]: removed this part to avoid confusion, I believe it's clear that it's about user's feature flag

**Review by IlyaIzr — APPROVED** [review]

**Pull request merged**